dialogue-apache-hc4-client/src/main/java/com/palantir/dialogue/hc4/ApacheHttpClientChannels.java

```diff
@@ -80,9 +82,10 @@
     private ApacheHttpClientChannels() {}
 
     public static Channel create(ClientConfiguration conf) {
```

Comment:
thoughts on deprecating/deleting this one at the same time? it leaks resources by never closing the CloseableClient, and we don't use it from WC...

Reply:
I think this should move to a test utility rather than main sources, in favor of deprecating this method but I'd prefer to do it in isolation -- looks like ~7 uses in test code that will need to be either updated or suppress the deprecation.